services/loggers/class-wc-ebanx-debug-logger.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use EBANX\Plugin\Services\WC_EBANX_Log;
+
+/**
+ * Log Checkout event data
+ */
+final class WC_EBANX_Debug_Logger extends WC_EBANX_Logger {
+	/**
+	 *
+	 * @param array $log_data data to be logged.
+	 * @param string $event event name to be logged.
+	 */
+	public static function persist( array $log_data = [], $event = null ) {
```

Comment:
```suggestion
	public static function persist( array $log_data = [], $event = null) {
```

Reply:
Why was this parameter added to this function? All loggers that extend this class have this parameter on the function declaration, but none of them use it